### DISCUSSION TITLE
es8388 audio codec ESP32 a1s

### DISCUSSION THREAD

**Discussion by noorino1977** [garygru/yummyDSP#16] [Q&A]
any one made it with  es8388 audio codec ESP32 a1s

**Comment by garygru** [comment]
The framework works fine with the old A1S based on the AC101. Didn't look into the differences to the new one. Anyhow, [this discussion ](https://github.com/marcel-licence/esp32_midi_sampler/discussions/50) looks interesting and may help how to adopt to the new codec.

**Reply by noorino1977** [reply]
I have added codec ES8388 for ESP32 A1S Audio Kit, v2.2 3378, maybe you can add it in your library

**Reply by noorino1977** [reply]
https://drive.google.com/drive/folders/1rntwYUo_KYdwBtG4bF9Gwngxfvag6EeU?usp=sharing

**Reply by garygru** [reply]
Cool, will integrate that! You're welcome to send a pull request. Thanks!

**Comment by noorino1977** [comment]
root your phone and create folder manually

Pada tanggal Jum, 11 Mar 2022 15.17, Gary Grutzek ***@***.***>
menulis:

**Comment by noorino1977** [comment]
i am sorry .. I sent a wrong email address

Pada tanggal Sab, 12 Mar 2022 18.34, Noors Channel Custom <
***@***.***> menulis:
